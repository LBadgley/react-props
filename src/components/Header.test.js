import React from 'react';
import { shallow } from 'enzyme';
import Dog from './Dog';

describe('header component', () => {
  it('renders a header with a doggo in it', () => {
    const wrapper = shallow(<Dog name="Ruby" age="2" weight="7.5 lbs"/>);
    expect(wrapper).toMatchSnapshot();
  });
});
