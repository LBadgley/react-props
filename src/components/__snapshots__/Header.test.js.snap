// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header component renders a header with a doggo in it 1`] = `
<dl>
  <dt>
    Name
  </dt>
  <dd>
    Ruby
  </dd>
  <dt>
    Age
  </dt>
  <dd>
    2
  </dd>
  <dt>
    Weight
  </dt>
  <dd>
    7.5 lbs
  </dd>
</dl>
`;
